controller/profile.go

```diff
@@ -152,6 +153,37 @@ func AssignPastAcademics(field *interfaces.ProfilePastEducation, education **stu
 
 }
 
+func AssignPastDiploma(field *interfaces.DiplomaDetails, education **studentModel.DiplomaDetails, forward bool) {
```

Comment:
Why not reuse AssignPastEducation?

Reply:
Since I created a new diploma struct AssignPastEducation was not compatible with it so I created AssignPastDiploma function.